Automattic/wp-calypso
# Use akismet plan constants to show benefits at checkout

This diff employs the recently added Akismet plan constants to show plan benefits at checkout and properly classify the plans as akismet plans.

Related to #

## Proposed Changes

* Adds additional `isAkismet...` functions to check for different plan types and show updated plan benefits at checkout for newly added plans.

## Testing Instructions

* Check out this branch locally or use the Calypso live link
* You will need to use the test store for this test plan
* Check all the new products in the checkout `/checkout/akismet/PRODUCT_SLUG`
* Make sure that the product benefits show in right-hand side of the checkout and that each plan shows the correct API limit as one of the product benefits
* Confirm that you can switch between one month, one year and two year subscription periods for each product

![Screenshot 2023-10-30 at 4 15 39 PM](https://github.com/Automattic/wp-calypso/assets/18016357/6587ea8d-d0b1-4b4e-9792-9be91fd43b19)

Product slugs to test:
- ak_pro5h_monthly
- ak_pro5h_yearly
- ak_pro5h_bi_yearly
- ak_bus5k_monthly
- ak_bus5k_yearly
- ak_bus5k_bi_yearly
- ak_ep15k_monthly
- ak_ep15k_yearly
- ak_ep15k_bi_yearly
- ak_ep25k_monthly
- ak_ep25k_yearly
- ak_ep25k_bi_yearly

## Pre-merge Checklist



- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?his
